src/test/resources/inlineLink5.txt

```diff
@@ -0,0 +1,143 @@
+'- javadoc
+   |- mainDescription
+   |  |- inlineTag
+   |  |  |- TOKEN[type: JAVADOC_INLINE_TAG_START, text: {@]
+   |  |  |- linkPlainInlineTag
+   |  |  |  |- TOKEN[type: LINKPLAIN_LITERAL, text: linkplain]
+   |  |  |  |- reference
+   |  |  |  |  |- qualifiedName
+   |  |  |  |  |  |- TOKEN[type: IDENTIFIER, text: java]
+   |  |  |  |  |  |- TOKEN[type: DOT, text: .]
+   |  |  |  |  |  |- TOKEN[type: IDENTIFIER, text: util]
+   |  |  |  |  |  |- TOKEN[type: DOT, text: .]
+   |  |  |  |  |  |- TOKEN[type: IDENTIFIER, text: Map]
+   |  |  |  |  |  '- typeArguments
+   |  |  |  |  |     |- TOKEN[type: LT, text: <]
```

Comment:
Why lt and but not lt literal? Same question with dot.

Is there a pattern we have when we use literal and not use it in the name?

Reply:
Personally if LITERAL would go completely away I would be very happy

Reply:
@mahfouz72 I don't see any conclusion here. Are we going to remove literal or use it more?

Reply:
@mahfouz72 please create an issue for this one also ^